src/internal/config.ts

```diff
@@ -144,6 +155,13 @@ export interface AWSConfigOptions {
      * @type {string}
      */
     endpoint?: string
+
+    /**
+     * RGW compatible.
+     *
+     * @type {boolean}
+     */
+    rgw?: booleaan
```

Comment:
```suggestion
    rgw?: boolean
```

Although, is this property required? We should aim to make this library work with any S3-compatible API, so adding some Ceph-specific options seems unnecessary.

Could we make this work by just overriding `endpoint`?